examples/src/main/scala/sttp/tapir/examples/booksExample.scala

```diff
@@ -56,6 +57,16 @@ import sttp.tapir.generic.auto.*
     val booksListingByGenre: PublicEndpoint[BooksQuery, String, Vector[Book], Any] = baseEndpoint.get
       .in(("list" / path[String]("genre").map(Option(_))(_.get)).and(limitParameter).mapTo[BooksQuery])
       .out(jsonBody[Vector[Book]])
+
+    // Optional value from serverLogic, responding with 404 when None
```

Comment:
let's create a stand-alone, separate example. Otherwise this one will become a monster-example ;)

Reply:
Done